frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal-enhanced.tsx

```diff
@@ -61,6 +62,7 @@ const DiskModalFirehoseComponent: React.FC<DiskModalFirehoseComponentProps> = (p
       volume={volumeWrapper}
       dataVolume={dataVolumeWrapper}
       onSubmit={onSubmit}
+      allowedBusses={new Set(DiskBus.getAll())}
```

Comment:
probably not here though as this is just the modal

Reply:
lets remove this line and let DiskModal take care of it for now

Reply:
First, this is just for now, there will be a followup. Second, this is DiskModal prop, it has to get the default value.

Reply:
fair enough, let's do it in a followup. Anyway the DiskModal could contain the default value by itself